manifests/m/Microsoft/NuGet/6.8.0/Microsoft.NuGet.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.NuGet
+PackageVersion: 6.8.0
+InstallerType: portable
+Commands:
+- nuget
+AppsAndFeaturesEntries:
+- DisplayName: NuGet CLI
+  DisplayVersion: 6.8.0.127

```

Comment:
```suggestion
  DisplayVersion: 6.8.0.131
```
Version needs to be updated
![image](https://github.com/microsoft/winget-pkgs/assets/88161975/4c1585d9-ad29-4902-b373-af868fa792ae)

Reply:
Cheers! @mdanish-kh